Public/Src/Tools/SymbolDaemon/Tool.SymbolDaemon.dsc

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import {Artifact, Cmd, Transformer} from "Sdk.Transformers";
+
+import * as BuildXLSdk from "Sdk.BuildXL";
+import * as Managed from "Sdk.Managed";
+import * as Deployment from "Sdk.Deployment";
+import { NetFx } from "Sdk.BuildXL";
+
+export declare const qualifier : BuildXLSdk.FullFrameworkQualifier;
+
+const symstoreX64Libs : Deployment.DeployableItem[] = getSymstoreX64Libs();
+
+@@public
+export const exe = !BuildXLSdk.isSymbolToolingEnabled ? undefined : BuildXLSdk.executable({
+    assemblyName: "SymbolDaemon",
+    rootNamespace: "Tool.SymbolDaemon",
+    appConfig: f`SymbolDaemon.exe.config`,
+    sources: globR(d`.`, "*.cs"),
+
+    references: [
+        importFrom("BuildXL.Cache.ContentStore").Hashing.dll,
+        importFrom("BuildXL.Engine").Scheduler.dll,
+        importFrom("BuildXL.Utilities.Instrumentation").Common.dll,
+        importFrom("BuildXL.Utilities.Instrumentation").Tracing.dll,
+        importFrom("BuildXL.Utilities").dll,
+        importFrom("BuildXL.Utilities").Ipc.dll,
+        importFrom("BuildXL.Utilities").Native.dll,
+        importFrom("BuildXL.Utilities").Storage.dll,
+        importFrom("BuildXL.Tools").ServicePipDaemon.dll,
+
+        importFrom("ArtifactServices.App.Shared").pkg,
+        importFrom("ArtifactServices.App.Shared.Cache").pkg,        
+        importFrom("Drop.RemotableClient.Interfaces").pkg,
+        importFrom("ItemStore.Shared").pkg,
+        importFrom("Microsoft.ApplicationInsights").pkg,
+        importFrom("Microsoft.AspNet.WebApi.Client").pkg,
+        importFrom("Microsoft.IdentityModel.Clients.ActiveDirectory").pkg,
+        ...BuildXLSdk.visualStudioServicesArtifactServicesSharedPkg,
+        importFrom("Microsoft.VisualStudio.Services.BlobStore.Client").pkg,
+        importFrom("Microsoft.VisualStudio.Services.Client").pkg,
+        importFrom("Microsoft.VisualStudio.Services.InteractiveClient").pkg,
+        importFrom("Newtonsoft.Json").pkg,
+        importFrom("WindowsAzure.Storage").pkg,
+
+        importFrom("Symbol.App.Core").pkg,
+        importFrom("Symbol.Client").pkg,
+        importFrom("Microsoft.Windows.Debuggers.SymstoreInterop").pkg,
+    ],
+    runtimeContent: [

```

Comment:
could just write on single line if you prefer:
`runtimeContent: getSymstoreX64Libx()`

Reply:
Since the exe will be evaluated once per qualifier (which in this case is only one :) you don't need to cache it in a const since it is used only once. If you are preparing for reuse, all fine.

---
In reply to: [313490839](https://github.com/microsoft/BuildXL/pull/727#discussion_r313490839) [](ancestors = 313490839)